installer/auter_installer.yml

```diff
@@ -109,6 +109,18 @@
         regexp: configsnap.*post-update
         line: /usr/sbin/configsnap -d /root -t auter-configsnap-$(date +%Y-%m-%d) -p post-update &> /root/auter-configsnap-$(date +%Y-%m-%d)/configsnap/post-update.compare
 
+    - name: create configsnap pre reboot script
+      tags:
+        - configure
+        - configsnap
+      lineinfile: 
+        create: yes
+        state: present
+        dest: /etc/auter/pre-reboot.d/99-configsnap-pre-reboot
+        mode: 0755
+        regexp: configsnap.*pre-reboot
+        line: 'if find /root/auter-configsnap-$(date +%Y-%m-%d)/configsnap/ -name "*.pre" &>/dev/null; then /usr/sbin/configsnap -d /root -t auter-configsnap-$(date +%Y-%m-%d) -p pre &> /root/auter-configsnap-$(date +%Y-%m-%d)/configsnap/pre-reboot.compare; else logger -p info -t auter "WARNING: Configsnap pre-apply files missing, running $0 for post-reboot diff"; fi'
```

Comment:
1) add "--silent" to the configsnap call
2) Remove the redirection for output "&> /root/auter-configsnap-$(date +%Y-%m-%d)/configsnap/pre-reboot.compare"